.github/workflows/cargo-deny-pr.yml

```diff
@@ -1,11 +1,17 @@
 name: cargo deny
+
 on:
   push:
     paths:
       - '**/Cargo.toml'
   pull_request:
     paths:
       - '**/Cargo.toml'
+
+concurrency:
+  group: cargo-deny-${{ github.ref }}
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow <- this is a neat suggestion for naming concurrency groups

It's fine in your setup because you used unique names everywhere. Up to you if you want to use `github.workflow` in the names.

Reply:
This is even better, thanks for the suggestion!